less/home-page.less

```diff
@@ -1 +1,94 @@
-// Home Content
\ No newline at end of file
+// Home Content
+
+.container {
+    width: 100%;
+    max-width: 880px;
+    font-size: 1.6rem;
+    margin: 0 auto;
+
+    .topImg {
+        width: 100%;
+        height: 240px;
+        margin-bottom: @marginBottom;
+    }
+    
+    h2 {
+        font-size: 2.4rem;
+        margin-bottom: @marginBottom;
+    }
+
+    p {
+        margin-bottom: @marginBottom;
+    }
+
+    .mainInfo {
+        display: flex;
+        flex-direction: column;
+        margin: 0 1%;
+        @media @phone {
+
+            margin: 0 2%;
+
+            .midImg2 {
+                display: none;
+            }
+
+            .midImg3 {
+                display: none;
+            }
```

Comment:
Is there a CSS short-hand to apply the same styling to two classes at once? Also, why use two classes that do the same thing?